### PR TITLE
Test fast copy

### PR DESCRIPTION
**Pull request by twicki**
## Purpose

The copy_corner 

## Code changes:

- Provide a list of relevant code changes and the side effects they have on other code.

## Requirements changes:

- Provide a list of any changes made to requirements, e.g. changes to files requirements*.txt, constraints.txt, setup.py, pyproject.toml, pre-commit-config.yaml and a reason if not included in the Purpose section (e.g. incompatibility, updates, etc)

## Infrastructure changes:

- Provide a list of changes that impact the infrastructure around running the code -- that is, changes to Makefiles, docker files, git submodules, or .jenkins (testing infrastructure changes). If Jenkins plans are also being manually changed, indicate that as well.

## Checklist
Before submitting this PR, please make sure:

- [ ] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [ ] Docstrings and type hints are added to new and updated routines, as appropriate
- [ ] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [ ] Unit tests are added or updated for non-stencil code changes

Additionally, if this PR contains code authored by new contributors:

- [ ] The names of all the new contributors have been added to CONTRIBUTORS.md

### CODE REVIEW

File: fv3core/stencils/basic_operations.py [hunk 1]
```diff
@@ -18,6 +18,17 @@ def copy_stencil(q_in: FloatField, q_out: FloatField):
         q_out = q_in
 
 
+def copy_defn(q_in: FloatField, q_out: FloatField):
```
Comment (rheacangeo) [inline fv3core/stencils/basic_operations.py:21]: nice -- I did this same thing with copy_stencil_method in master now, but I like your name better, would you mind replacing that with this?

File: fv3core/stencils/fvtp2d.py [hunk 2]
```diff
@@ -117,6 +119,31 @@ def __init__(self, namelist, hord):
         self.y_piecewise_parabolic_outer = YPiecewiseParabolic(
             namelist, ord_outer, self.grid.is_, self.grid.ie
         )
+        origin = self.grid.full_origin()
+        domain = self.grid.domain_shape_full()
+        ax_offsets = axis_offsets(spec.grid, origin, domain)
+        self._copy_full_domain = StencilWrapper(
+            func=copy_defn, origin=origin, domain=domain
+        )
```
Comment (rheacangeo) [inline fv3core/stencils/fvtp2d.py:127]: this also needs to extend to domain=self.grid.domain_shape_full(add=(0, 0, 1)), then everything will validate! woo nice job!

File: fv3core/stencils/del2cubed.py [hunk 3]
```diff
@@ -103,6 +107,26 @@ def __init__(self, grid):
...
+            domain=(self.grid.nid, self.grid.njd, self.grid.npz),
+            externals={
+                **ax_offsets,
+            },
+        )
+        self._copy_corners_y = StencilWrapper(
+            func=corners.copy_corners_y_stencil_in_out,
+            origin=(self.grid.isd, self.grid.jsd, 0),
+            domain=(self.grid.nid, self.grid.njd, self.grid.npz),
+            externals={
```
Comment (rheacangeo) [inline fv3core/stencils/del2cubed.py:125]: I know you are just using what was there already, but these can be self,.grid,full_domain() and self.grid.domain_shape_full()

File: fv3core/stencils/basic_operations.py (hunk 1 above)
Comment (twicki) [reply fv3core/stencils/basic_operations.py:21]: done

File: fv3core/stencils/fvtp2d.py (hunk 2 above)
Comment (twicki) [reply fv3core/stencils/fvtp2d.py:127]: thanks a lot!

File: fv3core/stencils/del2cubed.py (hunk 3 above)
Comment (twicki) [reply fv3core/stencils/del2cubed.py:125]: this should change in the oopified version.

**Comment by twicki** [comment]
closed in favor of https://github.com/VulcanClimateModeling/fv3core/pull/452

**Pull request closed without merging**